cli/packages/util/helper.go

```diff
@@ -298,3 +299,16 @@ func GenerateRandomString(length int) string {
 	}
 	return string(b)
 }
+
+func GenerateETagFromSecrets(secrets []models.SingleEnvironmentVariable) string {
+	sortedSecrets := SortSecretsByKeys(secrets)
+	content := []byte{}
+
+	for _, secret := range sortedSecrets {
+		content = append(content, []byte(secret.Key)...)
+		content = append(content, []byte(secret.Value)...)
+	}
+
+	hash := sha256.Sum256(content)
+	return fmt.Sprintf(`"%s"`, hex.EncodeToString(hash[:]))
+}
```

Comment:
this is probably something we'll want to add to our backend response in the future. I can see this being useful for other use-cases

Reply:
We actually already do this on the serverside, but I recall the ETag being unreliable and not working. So my solution to get around it is to just compute the Etag on the client side